core-android/build.gradle

```diff
@@ -17,7 +17,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
```

Comment:
why removing jcenter?

Reply:
jcenter got discontinued more than 2 years ago replacing it with mavenCentral